### ISSUE TITLE
NuxtJs - Can't load module

### ISSUE THREAD

**Issue by Geminii** [superman66/vue-highcharts#42]
Hi again, 
I try to load module like Exporting or Maps but it's not working. 

`import Exporting from 'highcharts/modules/exporting'
import Drilldown from 'highcharts/modules/drilldown'
import Highcharts from 'highcharts'
// Load Drilldown module
Drilldown(Highcharts);
// Load Exporting module
Exporting(Highcharts);`

On my project, i need to create Highcharts, Highmaps graphs. 
I try the same things on your projet vue-nuxt-demo and got the same problem to load modules. 
The import works but not the loading. 

How can i do this ? 
Maybe you can add some example to do this ?

Thanks a lot for your answer :)

**Comment by superman66** [comment]
I found it works, load Module in `mounted()`: 
```
  mounted() {
    Drilldown(Highcharts)
    Highcharts.setOptions({
      lang: {
        drillUpText: 'back',
      },
    })
  },
```
I also update demo in [vue-nuxt-demo](https://github.com/superman66/vue-nuxt-demo/blob/master/pages/index.vue).

**Comment by Geminii** [comment]
@superman66 i success to fix it too to define highcharts data. 
Load module map is ok but now i try to create an Highmaps but more difficult. If you have an example to do it, could be pretty cool :) 
Thanks :)

**Comment by superman66** [comment]
vue-highcharts seems to does't support HighMaps

**Comment by Geminii** [comment]
yep, i try to understand how it works. And it only works for line / bar etc. But to construct an Highmaps, another method need to be used. 
Could be nice to add a new feature to implement it like Highstock no ?

**Comment by superman66** [comment]
@Geminii I add highmaps support in the latest version.
And I also add highmaps demo in [vue-nuxt-demo](https://github.com/superman66/vue-nuxt-demo/blob/master/pages/index.vue)
